### PR TITLE
Feature/sync/508 status admin path

### PR DESCRIPTION
**Pull request by subigre**
## Changes

Update paths for Admin API, Management API and `/status` endpoint

- Update path for Status endpoint to `/rest/status`.
- Update base path for Admin API to` /rest/admin`.
- Update Actuator REST API base path to `/management`.

## Related issue

Resolves ehrbase/project_management#508

### CODE REVIEW

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/StatusController.java
Comment (jakesmolka) [inline rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/StatusController.java:39]: I'm not sure, but would it make sense to also move the file out of the `.../rest/openehr/...` scope now?

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminCompositionController.java
Comment (jakesmolka) [inline rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminCompositionController.java:44]: Historically, the file names with `Openehr[...]` were used to distinguish between the EhrScape and openEHR API. Since you removed it from the openEHR API, it might make sense to rename and/or to move the file out of the `.../rest/openehr/...` scope now.

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminContributionController.java
Comment (jakesmolka) [inline rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminContributionController.java:42]: Historically, the file names with `Openehr[...]` were used to distinguish between the EhrScape and openEHR API. Since you removed it from the openEHR API, it might make sense to rename and/or to move the file out of the `.../rest/openehr/...` scope now.

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminController.java
Comment (jakesmolka) [inline rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminController.java:37]: Historically, the file names with `Openehr[...]` were used to distinguish between the EhrScape and openEHR API. Since you removed it from the openEHR API, it might make sense to rename and/or to move the file out of the `.../rest/openehr/...` scope now.

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminDirectoryController.java
Comment (jakesmolka) [inline rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminDirectoryController.java:43]: Historically, the file names with `Openehr[...]` were used to distinguish between the EhrScape and openEHR API. Since you removed it from the openEHR API, it might make sense to rename and/or to move the file out of the `.../rest/openehr/...` scope now.

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminEhrController.java
Comment (jakesmolka) [inline rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminEhrController.java:42]: Historically, the file names with `Openehr[...]` were used to distinguish between the EhrScape and openEHR API. Since you removed it from the openEHR API, it might make sense to rename and/or to move the file out of the `.../rest/openehr/...` scope now.

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminTemplateController.java
Comment (jakesmolka) [inline rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminTemplateController.java:39]: Historically, the file names with `Openehr[...]` were used to distinguish between the EhrScape and openEHR API. Since you removed it from the openEHR API, it might make sense to rename and/or to move the file out of the `.../rest/openehr/...` scope now.

**Review by jakesmolka — COMMENTED** [review]
Looks good, but please see inline comments.

And also please let projects (like num-core) know that possible breaking changes are upcoming.

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminTemplateController.java
Comment (subigre) [reply rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminTemplateController.java:39]: Fixed

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/StatusController.java
Comment (subigre) [reply rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/StatusController.java:39]: Fixed

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminCompositionController.java
Comment (subigre) [reply rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminCompositionController.java:44]: Fixed

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminContributionController.java
Comment (subigre) [reply rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminContributionController.java:42]: Fixed

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminController.java
Comment (subigre) [reply rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminController.java:37]: Fixed

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminDirectoryController.java
Comment (subigre) [reply rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminDirectoryController.java:43]: Fixed

File: rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminEhrController.java
Comment (subigre) [reply rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminEhrController.java:42]: Fixed

**Review by jakesmolka — APPROVED** [review]
Hope you weren't waiting for my formal approval. Now with all the fixes all looks fine.

**Pull request merged**
